test/testSetup.ts

```diff
@@ -165,6 +185,83 @@ export class TestSetup {
     utilities.log('');
   }
 
+  public async verifyAllExtensionsAreRunning(): Promise<void> {
+    utilities.log('');
+    utilities.log(`${this.testSuiteSuffixName} - Starting verifyAllExtensionsAreRunning()...`);
+
+    // Using the Command palette, run Developer: Show Running Extensions
+    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.showRunningExtensions(workbench);
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Zoom Out', 2);
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Zoom Out', 2);
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Zoom Out', 2);
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Zoom Out', 2);
+    await utilities.pause(10);
+
+    // Verify CLI Integration extension is present and running
+    const coreExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-core'
+    );
+    expect(coreExtensionWasFound).toBe(true);
+
+    // Verify Apex extension is present and running
+    const apexExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-apex'
+    );
+    expect(apexExtensionWasFound).toBe(true);
+
+    // Verify Apex Replay Debugger extension is present and running
+    const ardExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-apex-replay-debugger'
+    );
+    expect(ardExtensionWasFound).toBe(true);
+
+    // Verify Apex Interactive Debugger extension is present and running
+    const isvExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-apex-debugger'
+    );
+    expect(isvExtensionWasFound).toBe(true);
+
+    // Verify SOQL extension is present and running
+    const soqlExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-soql'
+    );
+    expect(soqlExtensionWasFound).toBe(true);
+
+    // Verify Aura extension is present and running
+    const auraExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-lightning'
+    );
+    expect(auraExtensionWasFound).toBe(true);
+
+    // Verify Visualforce extension is present and running
+    const vfExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+      workbench,
+      'salesforcedx-vscode-visualforce'
+    );
+    expect(vfExtensionWasFound).toBe(true);
+
+    // // Verify LWC extension is present and running
+    // const lwcExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+    //   workbench,
+    //   'salesforcedx-vscode-lwc'
+    // );
+    // expect(lwcExtensionWasFound).toBe(true);
```

Comment:
The LWC extension does not start up properly on GHA, so it is omitted from the checks.  Only the LWC LSP E2E test uses this extension, so it is okay to not load it for all other E2E tests.